docs-ref-conceptual/azure-cli-extensions-overview.md

```diff
@@ -43,6 +45,22 @@ az extension add --source <URL-or-path>
 Once an extension is installed, it's found under the value of the `$AZURE_EXTENSION_DIR` shell variable. If this variable is unset, by default the value is `$HOME/.azure/cliextensions` on
 Linux and macOS, and `%USERPROFILE%\.azure\cliextensions` on Windows.
 
+### Install extensions automatically
+
+Azure CLI also has the ability to install an extension dynamically based on the command you run if the command is from an extension that is not installed yet. You can enable the feature through configuration:
+```azurecli-interactive
+az config set extension.use_dynamic_install=yes_prompt
```

Comment:
how about list the 3 options with description on behavior of each option

Reply:
Listed all 3 options.